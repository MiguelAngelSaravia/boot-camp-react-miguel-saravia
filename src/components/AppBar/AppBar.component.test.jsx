import React from 'react';
import {shallow} from '../../enzyme'

import CustomAppBar from './';
import AuthProvider from '../../providers/Auth';

describe('CustomAppBar', () => {
    const wrapper = shallow(
        <AuthProvider>
          <CustomAppBar
          />
        </AuthProvider>
      );
    
      it('Renders CustomAppBar', () => {
        expect(wrapper.exists()).toBe(true)
      });

      it('match snapshot CustomAppBar', () => {
        expect(wrapper.find('CustomAppBar')).toMatchSnapshot();
      });

      it('Return Component', () => {
        expect(wrapper.find('CustomAppBar').length).toBe(1);
      });
});
