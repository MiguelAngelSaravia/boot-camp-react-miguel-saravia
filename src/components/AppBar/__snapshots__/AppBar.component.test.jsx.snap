// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomAppBar match snapshot CustomAppBar 1`] = `ShallowWrapper {}`;
